ports/espressif/boards/lolin_c3_pico/mpconfigboard.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Scott Shawcroft for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+// Board setup
+
+#define MICROPY_HW_BOARD_NAME       "Wemos Lolin C3 Pico" // from Wemos MP
+#define MICROPY_HW_MCU_NAME         "ESP32-C3FH4" // from Wemos MP
+
+// From Wemos C3 Mini Schematic
+// https://www.wemos.cc/en/latest/_static/files/sch_c3_mini_v1.0.0.pdf
+// And MP Config
+// https://github.com/micropython/micropython/blob/master/ports/esp32/boards/LOLIN_C3_MINI
+// Status LED
+#define MICROPY_HW_NEOPIXEL         (&pin_GPIO7)
+
+#define CIRCUITPY_BOARD_I2C         (1)
+#define CIRCUITPY_BOARD_I2C_PIN     {{.scl = &pin_GPIO6, .sda = &pin_GPIO5}}
```

Comment:
`board.I2C()` should use the same pins as `board.SCL` and `board.SDA`, as on the S2 Pico. So I think you want to change these as below:
```suggestion
#define CIRCUITPY_BOARD_I2C_PIN     {{.scl = &pin_GPIO10, .sda = &pin_GPIO8}}
```